driver/driver.go

```diff
@@ -146,7 +151,7 @@ func parseFlags() (*driverFlags, error) {
 	flags.commonFlags = cli.RegisterCommonFlags()
 	flag.StringVar(&flags.manualTestMode, "manual-test-mode", "skip", "How to handle affected tests tagged manual. Possible values: run|skip")
 	flag.StringVar(&flags.targetPatternFile, "target-pattern-file", "", "If defined, stores the list of affected targets in the given file.")
-
+	flag.StringVar(&flags.bazelCommandOverride, "bazel-command-override", "auto", "Overrides the bazel command which runs build/test based on target type to run only build. Options: build, default behaviour will selected build/test based on target type")
```

Comment:
Isn't this really overriding only the Bazel subcommand used for running tests? The flag name suggests that it's more general than that.

Reply:
I've changed this to bazelSubCommandOverride. Does this sound more accurate to you?